newton/tests/test_mujoco_general_actuators.py

```
@@ -0,0 +1,567 @@
+                    newton_gainprm[world_newton_idx, :3],
+                    atol=1e-5,
+                )
+                np.testing.assert_allclose(
+                    mjw_biasprm[world, mj_idx, :3],
+                    newton_biasprm[world_newton_idx, :3],
+                    atol=1e-5,
+                )
+
+    def test_ordering_matches_native_mujoco(self):
+        """Test actuator ordering matches native MuJoCo loading."""
+        import mujoco  # noqa: PLC0415
```

Ruff (0.14.14)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)